src/payload.rs

```diff
@@ -25,45 +25,30 @@ use {
     base64::prelude::{Engine, BASE64_STANDARD},
     serde::{Deserialize, Serialize},
     solana_client::rpc_config::RpcSendTransactionConfig,
-    solana_sdk::transaction::VersionedTransaction,
+    solana_sdk::{pubkey::Pubkey, transaction::VersionedTransaction},
     solana_transaction_status::UiTransactionEncoding,
     std::str::FromStr,
     thiserror::Error,
 };
 
 #[derive(Debug, Default, Clone, PartialEq, Eq, Serialize, Deserialize)]
 #[serde(rename_all = "camelCase")]
-pub struct RpcSendTransactionConfigWithBlockList {
+pub struct RpcSendTransactionConfigWithPolicies {
     #[serde(flatten)]
-    pub config: Option<RpcSendTransactionConfig>,
-    /// base58-encoded pubkeys
-    pub blocklist_pdas: Option<Vec<String>>,
+    pub config: RpcSendTransactionConfig,
+    pub policies: Vec<Pubkey>,
```

Comment:
We should use `Option<Vec<String>>` not pubkey because we need to support base58 pubkey instead of just bytes